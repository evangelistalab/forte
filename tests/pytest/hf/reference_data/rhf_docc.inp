molecule {
Li 0.0 0.0 0.0
H  0.0 0.0 1.6
}

set scf_type pk
set basis cc-pVDZ
set docc [1,0,1,0]

energy('scf')
