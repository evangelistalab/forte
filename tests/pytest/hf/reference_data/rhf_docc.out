
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 undefined 

                         Git: Rev {master} ff26262 


    D. G. A. Smith, L. A. Burns, A. C. Simmonett, R. M. Parrish,
    M. C. Schieber, R. Galvelis, P. Kraus, H. Kruse, R. Di Remigio,
    A. Alenaizan, A. M. James, S. Lehtola, J. P. Misiewicz, M. Scheurer,
    R. A. Shaw, J. B. Schriber, Y. Xie, Z. L. Glick, D. A. Sirianni,
    J. S. O'Brien, J. M. Waldrop, A. Kumar, E. G. Hohenstein,
    B. P. Pritchard, B. R. Brooks, H. F. Schaefer III, A. Yu. Sokolov,
    K. Patkowski, A. E. DePrince III, U. Bozkaya, R. A. King,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, C. D. Sherrill,
    J. Chem. Phys. 152(18) 184108 (2020). https://doi.org/10.1063/5.0006002

                            Additional Code Authors
    E. T. Seidl, C. L. Janssen, E. F. Valeev, M. L. Leininger,
    J. F. Gonthier, R. M. Richard, H. R. McAlexander, M. Saitow, X. Wang,
    P. Verma, and M. H. Lechner

             Previous Authors, Complete List of Code Contributors,
                       and Citations for Specific Modules
    https://github.com/psi4/psi4/blob/master/codemeta.json
    https://github.com/psi4/psi4/graphs/contributors
    http://psicode.org/psi4manual/master/introduction.html#citing-psifour

    -----------------------------------------------------------------------


    Psi4 started on: Wednesday, 16 June 2021 02:16PM

    Process ID: 75200
    Host:       CHE-FEVANGE
    PSIDATADIR: /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule {
Li 0.0 0.0 0.0
H  0.0 0.0 1.6
}

set scf_type pk
set basis cc-pVDZ
set docc [1,0,1,0]

energy('scf')
--------------------------------------------------------------------------

Scratch directory: /tmp/

*** tstart() called on CHE-FEVANGE
*** at Wed Jun 16 14:16:59 2021

   => Loading Basis Set <=

    Name: CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1 entry LI         line    42 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/cc-pvdz.gbs 
    atoms 2 entry H          line    22 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2v
    Full point group: C_inf_v

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         LI           0.000000000000     0.000000000000    -0.200966416198     7.016003436600
         H            0.000000000000     0.000000000000     1.399033583802     1.007825032230

  Running in c2v symmetry.

  Rotational constants: A = ************  B =      7.47245  C =      7.47245 [cm^-1]
  Rotational constants: A = ************  B = 224018.52620  C = 224018.52620 [MHz]
  Nuclear repulsion =    0.992207270006250

  Charge       = 0
  Multiplicity = 1
  Electrons    = 4
  Nalpha       = 2
  Nbeta        = 2

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: CC-PVDZ
    Blend: CC-PVDZ
    Number of shells: 9
    Number of basis functions: 19
    Number of Cartesian functions: 20
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:               9
      Number of primitives:             30
      Number of atomic orbitals:        20
      Number of basis functions:        19

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 36290 doubles for integral storage.
  We computed 1035 shell quartets total.
  Whereas there are 1035 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 1.0272343532E-02.
  Reciprocal condition number of the overlap matrix is 3.3462107160E-03.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A1        10      10 
     A2         1       1 
     B1         4       4 
     B2         4       4 
   -------------------------
    Total      19      19
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:    -7.70182966811139   -7.70183e+00   0.00000e+00 
   @RHF iter   1:    -7.35033316901775    3.51496e-01   8.68811e-03 DIIS
   @RHF iter   2:    -7.40086413124468   -5.05310e-02   2.30816e-03 DIIS
   @RHF iter   3:    -7.40422733859302   -3.36321e-03   2.56012e-04 DIIS
   @RHF iter   4:    -7.40425597653011   -2.86379e-05   7.06190e-06 DIIS
   @RHF iter   5:    -7.40425598707951   -1.05494e-08   8.17043e-07 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A1    -2.550253     1B1    -0.133098  

    Virtual:                                                              

       2A1    -0.332016     3A1     0.003477     1B2     0.022531  
       4A1     0.034842     2B1     0.115646     2B2     0.130193  
       5A1     0.163386     6A1     0.193605     3B2     0.266818  
       3B1     0.275153     7A1     0.309850     1A2     0.309870  
       8A1     0.426274     9A1     0.544061     4B2     1.249178  
       4B1     1.258963    10A1     1.451263  

    Final Occupation by Irrep:
             A1    A2    B1    B2 
    DOCC [     1,    0,    1,    0 ]

  @RHF Final Energy:    -7.40425598707951

   => Energetics <=

    Nuclear Repulsion Energy =              0.9922072700062500
    One-Electron Energy =                 -11.4262156658484280
    Two-Electron Energy =                   3.0297524087626688
    Total Energy =                         -7.4042559870795088

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     1.5045

  Electronic Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.1081

  Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     1.6126     Total:     1.6126

  Dipole Moment: [D]
     X:     0.0000      Y:     0.0000      Z:     4.0987     Total:     4.0987


*** tstop() called on CHE-FEVANGE at Wed Jun 16 14:16:59 2021
Module time:
	user time   =       0.54 seconds =       0.01 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.54 seconds =       0.01 minutes
	system time =       0.04 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes

    Psi4 stopped on: Wednesday, 16 June 2021 02:16PM
    Psi4 wall time for execution: 0:00:00.67

*** Psi4 exiting successfully. Buy a developer a beer!
